app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int = 0,
+                 clean_mark: int = 0,
+                 brand: str = "") -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self, distance_from_city_center: float = 0,
+                 clean_power: int = 0,
+                 average_rating: float = 0,
+                 count_of_ratings: int = 0) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power and self.wash_single_car(car):
```

Comment:
The condition `car.clean_mark < self.clean_power` is checked twice: once here and again in the `wash_single_car` method. This redundancy could be avoided by ensuring the condition is only checked once.